### ISSUE TITLE
duplicateClassesStrategy options

### ISSUE THREAD

**Issue by tovshteyn** [melix/jmh-gradle-plugin#78]
Dear Cedric,

Let me start that there is nothing I can do about organizations having same classes in different libs.

So, I am only considering 3 other options:
include
exclude
warn

However, I would like to make sure to use only an option that complies with classpath rule: only first occurrence counts.

Based on the description in DuplicatesStrategy JavaDoc, it seems that only exclude does. However, I like a warn ability to report about duplicates.

Can you please confirm which one of the 3 complies with classpath rule?

Thank you

**Comment by vyazelenko** [comment]
@tovshteyn So you want something like "Exclude and warn"?

**Comment by tovshteyn** [comment]
Yes. Insuring standard classpath resolution rule (exclude) and be warned on the duplicate.

I was hoping that 'warn' does it because why would someone be warned on the duplicate and at the same time not follow standard classpath resolution rule.
